src/RemoteAPI/class-recommended-content.php

```diff
@@ -0,0 +1,54 @@
+<?php
+/**
+ * Recommended_Content class file.
+ *
+ * @package Parsely
+ * @since 3.2.0
+ */
+
+declare(strict_types=1);
+
+namespace Parsely\RemoteAPI;
+
+/**
+ * Recommended_Content
```

Comment:
Consider a more descriptive comment if possible. What does this class do or what is its use?